### ISSUE TITLE
docs: broken links in Apache APISIX dashboard document

### ISSUE THREAD

**Issue by fatihbm** [apache/apisix#9721]
### Current State

**Description of the current state/issue**:
There are many broken links on multiple pages. I can give an example of these. I think tests should be improved to prevent similar issues.
**Link to the page**:
An example situation can be seen in the third step in the link below.
https://apisix.apache.org/docs/dashboard/FAQ/#4-after-modifying-the-plugin-schema-or-creating-a-custom-plugin-in-apache-apisix-why-cant-i-find-it-on-the-dashboard

**Screenshots (if relevant)**:

### Desired State

**Description of the change you are proposing**:

### Would you like to fix this issue?

No. Please assign another contributor

**Comment by utkar-sh-ukla** [comment]
I'd like to take this up. @1502shivam-singh can you assign me this?

**Comment by utkar-sh-ukla** [comment]
@1502shivam-singh I've found an issue with our current approach to finding broken links, as many of the reported broken links are encountering CORS-related errors, not being truly broken. I tried adding a proxy, but it didn't fix the problem. However, I've found a straightforward solution using the npm package "[linkinator](https://github.com/JustinBeckwith/linkinator#readme)". We can seamlessly implement it into our workflow by integrating it directly into the website before the build process or by using the provided [Linkinator GitHub Action](https://github.com/marketplace/actions/linkinator). 
Can I proceed with this solution?
